src/plugins/region_map/server/services/opensearch_service.ts

```diff
@@ -3,8 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export default class OpensearchService {
-  constructor(esDriver) {
+import { ILegacyCustomClusterClient } from 'opensearch-dashboards/server';
+
+// eslint-disable-next-line import/no-default-export
+export default class OpenSearchService {
```

Comment:
Is there a reason we need a default export? Can't we just do a named export instead?

Reply:
Updating it to named export!